### PR TITLE
Fix various typos in Install page

### PR DESCRIPTION
**Pull request by maeriens**
Listed in #115

### CODE REVIEW

File: install.html [hunk 1]
```diff
@@ -15,62 +15,58 @@
...
+ you can add the dependency for that browser in your <i>pom.xml</i> file.
+ For example, you should add following dependency in your <i>pom.xml</i> 
+ file to run your tests only in Firefox:
 
-<pre>
-&lt;dependency&gt;
+<pre><code class=xml>&lt;dependency&gt;
   &lt;groupId&gt;org.seleniumhq.selenium&lt;/groupId&gt;
   &lt;artifactId&gt;selenium-firefox-driver&lt;/artifactId&gt;
   &lt;version&gt;2.53.1&lt;/version&gt;
```
Comment (shs96c) [inline install.html]: Aside: probably should be 3.x at this point :)

File: install.html [hunk 2]
```diff
@@ -15,62 +15,58 @@
...
 &lt;/dependency&gt;
-</pre>
+</code></pre>
 
 
 <h2>Using non browser specific JAR</h2>
 
-<p>More often than not you would need to run test
+<p> More often than not you would need to run test
```
Comment (shs96c) [inline install.html]: Pluralise `test`: `tests`

**Review by shs96c — APPROVED** [review]
Needs a rebase first, though.

**Comment by maeriens** [comment]
@shs96c Done -- firstly, I totally overlooked the Selenium version. Left it in 3.0.x for now. 
Secondly, I wrongly set the upstream as 'upstearm' so I thought there were no updates, my bad.

**Comment by maeriens** [comment]
The Travis build now passed on this one too!

**Comment by maeriens** [comment]
@andreastt can this be merged? I re updated it in it's 1 year anniversary 🎉

**Comment by andreastt** [comment]
I’m not a maintainer of Selenium anymore. Ping @lukeis.

**Comment by lukeis** [comment]
@andreastt I haven't been active in a while either :)

**Comment by andreastt** [comment]
Sorry, old habit (-;

**Pull request merged**